tests/test_project.py

```diff
@@ -2490,6 +2474,45 @@ def test_no_migration(self):
         assert len(migrations) == 0
 
 
+class TestSchemaMigration:
+    @pytest.mark.parametrize("implicit_version", [True, False])
+    def test_project_schema_version_migration(self, implicit_version):
+        from signac.contrib.migration import apply_migrations
+
+        with TemporaryDirectory() as dirname:
+            cfg_fn = os.path.join(dirname, "signac.rc")
+            with open(cfg_fn, "w") as f:
+                f.write(
+                    textwrap.dedent(
+                        """\
+                        project = project
+                        workspace_dir = workspace
+                        schema_version = 1"""
+                    )
+                )
+
+            config = read_config_file(cfg_fn)
+            print(config)
+            if implicit_version:
+                del config["schema_version"]
+                assert "schema_version" not in config
+            else:
+                config["schema_version"] = "0"
```

Comment:
Add a comment that this block is referring to schema version 0. It is a little confusing to have the string defined with version 1 only to revert here.

Reply:
I'll just define with version 0 to start with. No reason not to.